domains/nasirudeen.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "My personal page",
+  "repo": "https://github.com/iamnasirudeen",
+  "owner": {
+    "username": "iamnasirudeen",
+    "email": "iamnasirudeen@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://github.com/iamnasirudeen",

```

Comment:
This should be a host name. If you have a github pages repo, this should be `iamnasirudeen.github.io`. If you just want to redirect to your github repo, you need to remove of the cname entry.